### PR TITLE
SEXBLITTER

### PR DESCRIPTION
**Pull request by dankamongmen**
* Use Unicode 13 Legacy Computing sextant glyphs to create a 3:2 blitter, SEXBLITTER #1071 
* Add (k)eller demo #736

### CODE REVIEW

File: src/lib/blit.c [hunk 1]
```diff
@@ -448,6 +452,285 @@ quadrant_blit(ncplane* nc, int placey, int placex, int linesize,
...
+//fprintf(stderr, "start r: %u g: %u b: %u div: %u midbs: 0x%02x\n", r, g, b, div, mindiffbits);
+  for(unsigned shift = 0 ; shift < 6 ; ++shift){
+    if(mindiffbits & (1u << shift)){
+      // FIXME verify this
+      if(i < sizeof(fgcs) / sizeof(*fgcs)){
+        fgcs[i++] = *rgbas[shift];
+      }
+      const int rpos = bgr ? 2 : 0;
+      const int bpos = bgr ? 0 : 2;
+      r -= ((const uint8_t*)rgbas[shift])[rpos];
```
Comment (dankamongmen) [inline src/lib/blit.c:551]: This isn't safe on word-addressed architecutures. Use shifts.

File: src/lib/blit.c [hunk 2]
```diff
@@ -448,6 +452,285 @@ quadrant_blit(ncplane* nc, int placey, int placex, int linesize,
...
+strans_fold(unsigned* bitstring, unsigned bit, const uint32_t* rgba, bool bgr,
+            unsigned* rsum, unsigned* gsum, unsigned* bsum, unsigned* npopcnt){
+  if(ffmpeg_trans_p(bgr, ((const uint8_t *)rgba)[3])){
+    *bitstring |= bit;
+    return true;
+  }
+  const int rpos = bgr ? 2 : 0;
+  const int bpos = bgr ? 0 : 2;
+  ++*npopcnt;
+  *rsum += ((const uint8_t *)rgba)[rpos];
```
Comment (dankamongmen) [inline src/lib/blit.c:467]: Unsafe on word-addressed architectures. Replace with shifts.

**Review by joseluis — APPROVED** [review]

**Comment by joseluis** [comment]
awesome work man

**Comment by dankamongmen** [comment]
Still a bit of work necessary on this, but we're now always drawing the correct outline (i.e. transparency is 100%) and I just need figure out where we're reversing colors. It's gonna be AWESOME, much better than I expected.

**Comment by dankamongmen** [comment]
Gonna go ahead and merge this. It's pretty much working -- it oughtn't break anything, anyway -- and there are some important other fixes in there. Also, this PR is getting unwieldly.

It's not quite ready yet, but it's just about there.

**Comment by dankamongmen** [comment]
> awesome work man

thanks homie! gonna drop a gem on 'em shortly.

[let's go](https://www.youtube.com/watch?v=Sgm24qu2GzQ)

**Pull request merged**